CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Print values that are not object specifiers with `clojure.core/prn`.
```

Comment:
It would be nice if we could print always EDN-readable values. So we could print object specifiers as a real string as well (with quotes). Might be worth mentioning that this is only done with `-e`.

Reply:
I usually insert the issue number that was fixed in the changelog as well.

Reply:
Yes! I meant to ask you about this! One thing we could consider would be to add a tagged literal for object specifiers. That way they could be re-read after having been printed. I think I prefer that to printing them as strings; I'm a bit worried that doing that might confuse people. They aren't strings after all.

What do you think?

Reply:
We can start with strings. Let's not invest in complicated logic until it's really necessary.

Reply:
Oh yes, tagged literal might be good as well.

Reply:
I thought you also wanted to support reading them back in. Not sure about that :).

Reply:
Yeah, might be worthwhile to wait on supporting reading them in. There might be subtleties to that.

Reply:
Thoughts on the tag? Ideas:

* `#object-specifier`
* `#os`
* `#obb/os`
* `#jxa/os`
* …

Reply:
In either case I would go for a namespaced one to avoid conflicts with others. `org.babashka.obb/object-specifier` is a safe bet.

Reply:
Or something along those lines.

Reply:
```
❯ obb -e '(js/Application "Safari")'
#org.babashka.obb/object-specifier "Application(\"Safari\")"
```

Reply:
How's that look?

Reply:
That looks frigging awesome :)